src/App.tsx

```diff
@@ -8,37 +8,39 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
-import { getUserPosts } from './api/posts';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { setAuthor } from './features/author';
 import { User } from './types/User';
+import { clearPosts } from './features/posts';
+import { setSelectedPost as setSelectedPostAction } from './features/selectedPost';
```

Comment:
```suggestion
import { setSelectedPost } from './features/selectedPost';
```
And lint error won't be a problem anymore
Generally - it was suggested to remove `// disable lint rule`. In such case you should try to fix the error - not just silently ignore it 🙂